timeline_data.json

```diff
@@ -1,5 +1,18 @@
 {
   "items": [
+    {
+  "date_str": "2023-06-07",
+  "title": "Data Dumps Removed",
```

Comment:
The previous data dumps weren't removed - only the scheduled script was disabled. The title needs rewording to make this clearer with the slug updated along with it.